packages/@cdktf/hcl2cdk/lib/index.ts

```diff
@@ -301,10 +301,14 @@ export function getTerraformConfigFromDir(importPath: string) {
   return fileContents.join("\n");
 }
 
+type CdktfJson = Record<string, unknown> & {
+  terraformProviders: any[];
+  terraformModules: any[];
```

Comment:
nit: they could also be undefined / not set for the input, but they won't be when they're outputted, right?